samples/tutorial/Tutorial3.md

```diff
@@ -221,7 +221,9 @@ object TodoEditWorkflow : StatefulWorkflow<EditProps, State, Output, TodoEditScr
 
 We want the todo edit screen to be shown when a user taps on an item on the todo list screen. To do this, we will modify the todo list workflow to show the edit screen when we are editing.
 
-The `TodoListWorkflow` will now occasionally need to render two screens instead of just the one. Its parent workflow will add the one or two screens to the backstack it constructs. Since `TodoListWorkflow`'s rendering type is `Any`, it can just return a list of screens. However, the parent workflow needs to know about the list, so it can pull the screens out and add them to its backstack. We'll change the rendering type from `Any` to `List<Any>`:
+The `TodoListWorkflow` will now occasionally need to render two screens instead of just the one. Its parent workflow `RootWorkflow` will add the one or two screens to the backstack it constructs. We need to update `TodoListWorkflow`'s rendering type to `List<Any>` so that it can return a list of screens, instead of just `TodoListScreen`. The parent workflow needs to know about the list, so it can pull the screens out and add them to its backstack. 
```

Comment:
`TodoListWorkflow` 's render type is not `Any`. In tutorial 2 we still had it as `TodoListScreen`